L/libclangex/build_tarballs.jl

```diff
@@ -8,34 +8,42 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 
 name = "libclangex"
-version = v"0.1.8"
+repo = "https://github.com/Gnimuc/ClangCompiler.jl.git"
+version = v"0.2.0"
 
-llvm_versions = [#=v"11.0.1",=# v"12.0.1", v"13.0.1", v"14.0.5"]
+llvm_versions = [v"18.1.7"]
 
-# Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/Gnimuc/libclangex.git", "c22e5eadc5f05b2f174d49c8e2fc5fcb75c3fbdb")
+    GitSource(repo, "7d599c3f41a183aae68e53f510915d64bbad3da7")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir
-cd libclangex/
+cd ClangCompiler.jl/deps/ClangExtra
+
+if [[ "${bb_full_target}" == x86_64-apple-darwin* ]]; then
+    # LLVM 15+ requires macOS SDK 10.14.
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    popd
+fi
+
 mkdir build && cd build
 cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+     -DCMAKE_CROSSCOMPILING:BOOL=ON \
```

Comment:
CMake should be autodetecting this from our toolchain file. Are you seeing issues where this isn't happening?

Reply:
@imciner2 fixed.